src/Learners/Recommender/MatchboxRecommender.cs

```diff
@@ -70,22 +70,6 @@ public static class MatchboxRecommender
 
         #region .NET binary deserialization
         
-        /// <summary>
-        /// Deserializes a Matchbox recommender from a file.
-        /// </summary>
-        /// <typeparam name="TInstanceSource">The type of a source of instances.</typeparam>
-        /// <typeparam name="TUser">The type of a user.</typeparam>
-        /// <typeparam name="TItem">The type of an item.</typeparam>
-        /// <typeparam name="TRatingDistribution">The type of a distribution over ratings.</typeparam>
-        /// <typeparam name="TFeatureSource">The type of a feature source.</typeparam>
-        /// <param name="fileName">The file name.</param>
-        /// <returns>The deserialized recommender object.</returns>
-        public static IMatchboxRecommender<TInstanceSource, TUser, TItem, TRatingDistribution, TFeatureSource>
```

Comment:
These public methods have nothing to do with binary formatter.  They should be kept but migrated to text format.